Update ja_JP.lang

tento64 opened the pull request:
Fixed translation errors.
Some translations have been changed.
Add translation.

karakufire commented:
At lines 34, 37-39, 43 and 55:
I recommend "溶融" as the translation of "molten"
These fluids are metals molten by high-temperature, so I guess "溶融" is appropriate for translation of "molten".

At lines 26, 32, and  42:
'Blazing pyrotheum,' 'Gelid cryotheum' 'Zephyrean aerotheum,' and 'Tectonic Petrotheum are fluid of 4 elementals. I translated them with aware the yinyang-pairs of each elementals: "pyro-" and "cryo-", "aero-" and "petro-".
I request you to consider this background.

tento64 commented:
"溶融" is a technical term, so I thought "溶けた" was easier to understand, so I chose "溶けた"

Regarding the dust of the four major elements, there was an augment called tectonic initiator for thermal expansion, so I thought that the reason was that the "地殻変動起動装置" was easy to understand, and I decided to translate the term "tectonic" Changed to "地殻変動の".
And the word "地動" comes to mind in the Heliocentrism.

Since I was aware of word pairs as well, petrotheum's modifier was no longer two letters, so I changed "清風" into a longer word, "そよ風のような".

Regarding gelid, when translating Redstone Repository Revolved, I thought it would be better to use shorter words and changed it to "極寒". But I think it would have been nice to keep it "凍て付く".

「溶融」は、専門的な用語なので、私は「溶けた」の方が分かりやすいかなと思って、「溶けた」にしました。

4大元素のダストに関しては、サーマルエキスパンションの方に、tectonic initiatorというオーグメントがあったため、私はその訳は「地殻変動起動装置」が分かりやすいと考え、私は「テクトニック」の訳語を「地殻変動」に変更しました。
それと「地動」という単語は、地動説の方が浮かんでしまう。

私も単語のペアを意識していたので、ペトロジウムの修飾語が二文字でなくなったため、「清風」を長めの文字数の言葉である「そよ風のような」に変えました。

gelidに関しては、Redstone Repository Revolvedを翻訳する際、短めの単語の方がいいかと考え、「極寒」に変えました。ですが、「凍て付く」のままでも良かったかなとも思います。

karakufire commented:
[JP]
私のコメントに対し反応いただきありがとうございます。

1. 「溶けた」では、熱・化学・コロイド分散といった溶解自称を内包する曖昧な表現ですので、"molten"における、融点に達して液相へ相転移したものに対する訳としては不適切に思います、故に今の訳のほうが適していると考えます。

2. 単語のペアを同じように意識するのならば、例えば、"Aero-"を「そよ風のような」とし続けるならば、「～のような」にするほうが、自然にペアを感じられると思います。
ここは私も訳に悩んだ箇所です。ペア同士で統一性を満たせれば、私としては文句はございません。

3. "gelid"の訳については、対となる"blazing"の訳である「燃え盛る」をそのままにしていることから、2.で述べた要請に従ってそのままか、もしくは類似する構成の形容詞に書き換える必要があると思います。また、あなたが述べたRedstone Repostiry Revolvedについてですが、あなたによる日訳PR実績が認められないため、正当性がありません。

[EN]
Thank you for responsing to my comments.

1. 「溶けた」 is not suitable for translation of "molten", it's included meaning "by heat", "by chemical" and "by colloidal" too. So I recommend current's.

2. It's more natural for feeling to translate "tectonic" likes as 「～のような」 if you affiring translation "Aero-" as 「そよ風のような」. I had worried translating this sentenses too. I have no opinion if you unify translation between the pair.

3. About "gelid", you should keep as-is or suggest another similar abjective therefore you keep 「燃え盛る」of translation "blazing", the paired with cryotheum by 2. request.
And no your translation PR at "Redstone Repository Revolved," so no correctness in your insisting.

tento64 commented:
【原文】
moltenはmelt（溶ける）の過去分詞でもあるので、「溶けた」としました。
例えばニッケル溶液やニッケルの懸濁液などの表記なら化学的な溶解の意味になりますが、
単純に「溶けたニッケル」などのように金属に「溶けた」と付ける場合、溶鉱炉などのイメージも相まって、多くは高温によるものとイメージされます。
ただ、私が訳語を「溶融」でなく、「溶けた」とした理由は難解な用語よりも、日常的な単語の方が、砕けた感じがしてこのモッドにとっつき易いかなと思ったからです。

２．原文の方が、例えば現在分詞で統一されているというわけでないので、それほど統一に拘らなくてもいいと私は思います。それと「そよ風のような」は「地殻変動の」が四文字熟語で硬い印象を受けるため、その反対にひらがなでふわふわした表現の方があっているかなと思ったため採用しました。

３．Redstone Repostiry Revolvedは翻訳するならThermal foundationと訳語を統一しようと考えていただけで、実行には移していません。ただ、こちらは、「凍て付く」のままで良かったかなと私もおもっています。
【EN】
Since molten is a past participle of melt（溶ける）, the translation was changed to "溶けた".
For example, notation such as "ニッケル溶液" or "ニッケルの懸濁液" means chemical dissolution,
When simply saying "溶けた" to a metal, such as "溶けたニッケル", the image of a blast furnace, etc. is also combined, and most are thought to be due to high temperatures.
However, I chose the word "溶けた" instead of "溶融" because I thought that everyday words seemed more crumbled and more accessible to this mod than esoteric terms.

2. The original text, for example, is not currently unified with participles, so I think it is not necessary to be so united.I chose "そよ風のような" because "地殻変動の" has a hard impression with four-letter idioms, and on the contrary I thought that there was a more fluffy expression in hiragana.

3. Redstone Repostiry Revolved was just thinking about unifying the translation with the Thermal foundation when translating, but it has not been implemented.However, I think that it would be nice to keep it "凍て付く".

karakufire commented:
[JP]
あなたの視点をお教えいただきありがとうございます。
以下は私の持論でございます。ご一読ご一考くだされば幸いです
1. 私としてはとっつきやすさよりは原語の正確性、ニュアンスを第一に考えております。平易な言葉に置き換えて、もとのニュアンスを損なうのは無粋に感じます。
2. ほとんどの訳は既に[Minecraft Japan WikiのThermal Expansionの解説ページ](https://w.atwiki.jp/minecraft/pages/1201.html)の方でも使用されており、安易な訳の変更はユーザーの混乱を招く虞れがあります。原語が変更されたときか、対象の挙動が大きく変更されたとき以外はするべきでは無いでしょう。

上述の2点は、Thermal Foundationだけではなく、CoFHのMod全体に及ぶとおもいます。
なにか適切な場所で議論して、妥結点を探っていきたいと思います(CoFH/Feedbackのissueが無くなったので)。

[EN]
I understand your viewpoint to translation. Thank you.
Followings are my opinion. I would be glad if you read them and consider again.
1. I think accuracy and nuance of original words first, than easiness for doing. I feel it's boorish that losing the original nuances in plain language.
2. Mostly yranslations are already used in [Minecraft Japan Wiki's Thermal Expansion user-description](https://w.atwiki.jp/minecraft/pages/1201.html) pages. Thoughtless existing-translations changing risks confusing users. You should not do unless original has changed or behavior has changed significantly.

The 2 opinion I said is not only about Thermal Foundation JP-translation, all mods created by CoFH.
I'd like to discuss you in suitable location somewhere, because issue pages of CoFH/Feedback has gone before I knew 😢.

tento64 commented:
私が検索をしてみると、溶融鉄という単語は専門的な記事で使用され、溶けた鉄という単語はニュースなど、一般的な記事で使用されています。そのため、口語で使用する場合、溶けた鉄の方が圧倒的に伝わりやすいです。口語で話す場合、「溶融」では説明の手間を要します。
また、「melt」は専門的な印象の無い一般的な単語ですので、一般的な単語に対する対訳は同じく一般的な単語にすべきだと私は思います。
それと、「溶融」という単語は、「溶ける」に近い意味を持つ「溶」という文字と、「融合」に近い意味を持つ「融合」という文字で構成されています。

wikiはあまり編集されない記事や項目もあるため、権威的なものではないと私は思っています。

When I did a search, the word "溶融鉄" was used in specialized articles, and the word 溶けた鉄 was used in general articles, such as news. Therefore, when used in spoken language, "溶けた鉄" is easier to convey.When speaking in spoken language, "溶融" takes time to explain.
Also, since "melt" is a general word that does not have a professional impression, I think that a translation for a general word should also be a general word.
In addition, the word "溶融" is composed of the word "溶", which has a meaning similar to "melt", and the word "融", which has a meaning similar to "fuse".

I don't think the wiki is authoritative, as there are articles and items that are not often edited.

karakufire commented:
[JP]

以下は貴方の返答についての回答です。
1. 「溶融」では専門的、「溶けた」では一般的、というあなたの主張ですが、その主張の根拠を私でも確認できる手段はなにか、検索サイトやニュースサイト、検索語句を具体的にお教えいただけますか?

2. 字義的に論ずるならば、「融」の字も

    > とける。とかす。固体が液体になる。

    出典: [融|漢字一字|漢字ペディア](https://www.kanjipedia.jp/kanji/0006891000)

    とありますし、「融合」の熟語だけを挙げて「融」の字の意味を説くのは乱暴に思います。

3. 確かにそのwikiは更新されないページもありますが、先程挙げたページでも閲覧数は2020年2月18日20時時点で合計60万を超え、それぞれのThermal Foundation、Thermal Expansionのそれぞれの1.12.2版の解説ページもそれぞれ合計1万を超えています。これも、そのwikiには更新されないページが多いからと権威、影響力が無いと言い切るのは些か早計じゃ無いでしょうか。ただ、そのページ載ってないアイテムの訳については無関係とします。

以下は貴方のPRに対する他の部分についての私からの評価です。
* 163, 164, 210, 211行目
    * アクア・チャウ、フィト・グローについては、ある種商品名のような名前の付け方らしく、それらしさを感じさせる訳にしていました。
* 48行目 励起→不安定化
    * 単語的にはそれであっています、両方とも液体であるのを感じないので、私も悩んだところです。ただ、やはりwikiには「励起～」として掲載されているので十分考慮願います。また、Googleのサジェストでも励起レッドストーンの語句で出てきます。
* 192～207行目 顔料→色素
    * 色を染めるためのものですので、変更しないほうが良いと考えられます。あるいは、「染料」を提案します。
* 281行目 ダイヤの歯車→ダイヤモンドの歯車
    * Minecraftにおける「Diamond Sword」などの語句はすべて「ダイヤの～」で統一されています。それに準じるべきでしょう。
* 301行目 樹脂→松やに
    * 52行目と語句の統一がなされていません。
    * 松ヤニは文字通り松(厳密にはマツ属)から取れるものを指します。ResinはMinecraftの松以外からでも得られるため、「松ヤニ」へ変更するべきではありません。
* 682行目
    * "consult"の語義に「閲覧する」はありません。例えば、研究社のルミナス英和・和英辞典に拠れば、
        > 3 〈辞書〉を引く, 〈参考書など〉を調べる.

        とあります。  

追記: 貴方の他のCoFH ModsへのPRについても議論を行いたいので、今後はCoFHのdiscordのtranslationdiscussionチャンネルで行いませんか。

[Team CoFHのページ](http://teamcofh.com/)にそのdiscordのリンクがあります。

----

[EN]

Followings are my reply to yours.
1. On your opinion you said that "溶融" is expert, "溶けた" is generic; could you tell me the concrete way to examine grounds yours: searching or news web pages, phreases to searching.
2. If discuss focusing to meanings of kanji, also "融" means "melting" because
    > とける。とかす。固体が液体になる。

    authority: [融|漢字一字|漢字ペディア](https://www.kanjipedia.jp/kanji/0006891000)

    And it's too coarse explaining "融" with only examples "融合."
3. Surely this wiki has pages not mainttained a long time. But the page I picked has had over 600,000 total accesses as of 2020-02-18 20:00 Japan time. And, each descriptions of the Thermal Foundation's and Thermal Expansion's for MC1.12.2 also has over 10,000 accesses as the time. I think you're jumping to conclusion.
   
Next, followings are my reviews about your PR excludes about molten metals.
*  line 163, 164, 210, 211
   *  I  remember "aqua-chow" and "phyto-gro" is named like a product name, also I translated them likes it.
*  line 48 "励起"→"不安定化"
   *  Sure, it's right to literally. I had thought hard which one to adopt when I do it. But now, "励起" has appeared in the wiki, and also Google search suggests "励起レッドストーン."
*  line 192-207 "顔料"→"色素"
   *  These are for DYEING. It's fine as-is. Else I suggest "染料."
*  line 301 "樹脂"→"松やに"
   *  Not unified with at line 52.
   *  "松やに" (en: Pine resin) is only from *Pinus*. This item "Resin" can be got NOT ONLY Minecraft's "松"(in English lang files: Spruce). Should not change to "松やに."
*  line 682
   *  "Consult" does not means "read" or "browse." For example, according to 研究社のルミナス英和・和英辞典:
        > 3 〈辞書〉を引く, 〈参考書など〉を調べる.

P.S. I'd like to discuss with you about other your PRs to CoFH mods, let's discuss at "translationdiscussion" channel of CoFH Discord from now on.
[Tean CoFH official page](http://teamcofh.com/) has the link to discord.

tento64 commented:
分かりました。
Discordはほとんど使ったことがないですが、試してみます。

I understand.
I have never used Discord, but I will try it.
